CMakeLists.txt

```diff
@@ -47,6 +49,28 @@ set(H3_SOVERSION 1)
 
 project(h3 VERSION ${H3_VERSION})
 
+set(H3_COMPILE_FLAGS "")
+set(H3_LINK_FLAGS "")
+if(NOT WIN32)
+    # Compiler options are set only on non-Windows, since these options
+    # are not correct for MSVC.
+    list(APPEND H3_COMPILE_FLAGS -Wall)
+
+    list(APPEND H3_COMPILE_FLAGS $<$<CONFIG:Debug>:-g -gdwarf-2 -g3 -O0 -fno-inline -fno-eliminate-unused-debug-types>)
```

Comment:
👍

Reply:
Please remove `-O0` and `-g`. They don't serve the same purpose here as in the toolchain file.

Reply:
I see `-g` is in the default flags for Debug, but I don't see `-O0`. What is the reason for removing that option?